lib/launchers/openstack10.rb

```diff
@@ -687,6 +687,29 @@ def assign_ip(instance)
       end
     end
 
+    def allocate_floating_ip(network_name)
+      request_url = self.os_network_url + "/v2.0/floatingips"
+      method = "POST"
+      network = get_networks.find {|n| n["name"] == network_name}
+      unless network
+        raise "could not find network in current tenant."
+      end
+      network_id = network["id"]
+      payload = {
+        floatingip: {
+          floating_network_id: network_id,
+          project_id: self.os_tenant_id,
+          description: "IP allocated automatically by BushSlicer"
```

Comment:
project name is no longer BushSlicer, we just didn't figure out yet how to `sed` module/class names

Reply:
We can write it as `IP allocated automatically by OpenShift verification tests.`